devops/ci/single_node_deploy/Jenkinsfile_Deploy

```diff
@@ -77,13 +77,24 @@ node {
                 sshCommand remote: remote, command: """
                     yum install -y yum-utils
                     yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/3rd_party/"
-                    yum install --nogpgcheck -y python3 python36-m2crypto salt salt-master salt-minion
-                    rm -rf /etc/yum.repos.d/*3rd_party*.repo
                     yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/cortx_iso/"
+                    
+                    cat <<EOF >/etc/pip.conf
+                    [global]
+                    timeout: 60
+                    index-url: $CORTX_RELEASE_REPO/python_deps/
+                    trusted-host: $(echo $CORTX_RELEASE_REPO | awk -F '/' '{print $3}')
```

Comment:
the same question

Reply:
Yes @andkononykhin2, it's a new requirement that follows the cortx-preq rpm from RE.